.gitignore

```diff
@@ -126,3 +126,4 @@ results/
 
 # tests
 test_callback.log
+venv/
```

Comment:
This looks like an unrelated change, let's drop it. Pro tip in case you're not aware of it already: if you use `virtualenv` (or `uv venv`) to create your virtual environments instead of `venv`, you'll get a `.gitignore` file inside it and you won't need to git-ignore this directory 😉